<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindAndReplaceDialog</class>
 <widget class="QDialog" name="FindAndReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Find:</string>
           </property>
           <property name="buddy">
            <cstring>textToFind</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="textToFind"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="replaceLabel">
           <property name="text">
            <string>R&amp;eplace with:</string>
           </property>
           <property name="buddy">
            <cstring>textToReplace</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="textToReplace"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>D&amp;irection</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="upRadioButton">
            <property name="text">
             <string>&amp;Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="downRadioButton">
            <property name="text">
             <string>&amp;Down</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>&amp;Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="caseCheckBox">
            <property name="text">
             <string>&amp;Case sensitive</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="wholeCheckBox">
            <property name="text">
             <string>&amp;Whole words only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
